gecoscc-installer.sh

```diff
@@ -177,38 +178,44 @@ NO)
   esac
 fi
 
-
-echo "Adding EPEL repository"
+#echo "Adding EPEL repository"
 if ! rpm -q epel-release-6-8.noarch; then
-    rpm -ivh http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+#if ! rpm -q epel-release-7-9.noarch; then
+    rpm -ivh --nosignature http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+    #rpm -ivh --nosignature https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm
 fi
-echo "Installing python-devel and pip"
-install_package python-devel 
-install_package python-pip
+echo "Installing python2.7"
+yum install -y centos-release-SCL
```

Comment:
You need to install `centos-release-scl`.

From the [centos wiki](https://wiki.centos.org/AdditionalResources/Repositories/SCL):

> Gaining Access to SCLs

> In order to gain access to SCLs for CentOS, you need to install the CentOS Linux Software Collections release file. It is part of the CentOS Extras repository (x86_64 only) and can be installed with this command:

> yum install centos-release-scl